ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/DbDispatcher.scala

```diff
@@ -60,8 +60,8 @@ private class DbDispatcherImpl(
           logger.error(s"got an uncaught exception on thread: ${thread.getName}", t))
         .build()))
 
-  override def executeSql[T](sql: Connection => T): Future[T] =
-    sqlExecutor.runQuery(() => connectionProvider.runSQL(conn => sql(conn)))
+  override def executeSql[T](description: String)(sql: Connection => T): Future[T] =
```

Comment:
Same as above: if we are planning to access this only if trace logging is enabled, it could make sense to pass this by name.

Reply:
Addressed in 310fa9d95.